src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+    tag: 'chi-expansion-panel',
+    styleUrl: 'expansion-panel.scss',
+    scoped: true
+})
+export class ExpansionPanel {
+    /**
+     * to set expansion panel to 'done' state
+     */
+    @Prop({ reflectToAttr: true }) done?: boolean;
+
+    /**
+     * to set expansion panel to 'active' state
+     */
+    @Prop({ reflectToAttr: true }) active?: boolean;
+
+    /**
+     * to set expansion panel to 'disabled' state
+     */
+    @Prop({ reflectToAttr: true }) disabled?: boolean;
+
+    /**
+     * to set a step number next to the title of the panel
+     */
+    @Prop({ reflectToAttr: true }) step?: string;
+
+    /**
+     * to set the title of the panel
+     */
+    @Prop({ reflectToAttr: true }) eptitle: string;
```

Comment:
`title` would be much better than `eptitle`. I know stencil compiler complains about using reserved attribute names. Can you check if there is a workaround to use title here? @srinath-koneru, do you know a way? In the case there is no way to do it, I think it would be better to look for another name. In the worse scenario, epTitle (ep-title for the attribute) would be a solution. So: `title` > `good name you can find` > `epTitle` > `eptitle`

Reply:
@SergioQCL @dani-cl-madrid How about subject, entitle, name or even heading or let's ask @mattnickles .

Reply:
I like heading. In general I prefer single words over composed ones.

Reply:
@mattnickles We need to find an attribute name for the header of the Expansion Panel component. We cannot use `title` because it is an HTML reserved word. Subject, entitle, name or heading are some alternatives. Sergio likes heading. Any preference from you? Thank you!